pkgs/development/tools/fsautocomplete/default.nix

```diff
@@ -20,8 +27,8 @@ buildDotnetModule rec {
       --replace TargetFrameworks TargetFramework \
   '';
 
-  dotnet-sdk = with dotnetCorePackages; combinePackages [ sdk_6_0 sdk_7_0 sdk_8_0_2xx ];
-  dotnet-runtime = dotnetCorePackages.sdk_8_0_2xx;
+  dotnet-sdk = with dotnetCorePackages; combinePackages [ sdk_6_0 sdk_7_0 sdk_8_0_3xx ];
```

Comment:
Would this work with 8_0_1xx?  at least that's guaranteed to be maintained.

Reply:
yup, this seems to build with 8.0.1xx as well (no idea if it still works though, there *probably* was a good reason for explicitly using 8.0.2xx here? the commit that changed it is 6ee3c71201aa)

should we revert this to `sdk_8_0` then?

Reply:
it was pinned to 2xx because it didn't build on 3xx.  maybe the same thing your patch fixes?

I guess we should either put it back to 8_0, with the patch, and hope that it continues to build on the latest sdk, or pin it to 1xx and drop the patch.  I don't really have a strong opinion on that.

Reply:
alright, changed back to `sdk_8_0` now and builds fine. the patch is also required on 1xx anyway, so it doesn't really matter either way.